### PR TITLE
miscellaneous small changes

### PR DESCRIPTION
**Pull request by docrostov**
since we have a major release i don't think it is remotely essential to get this out quickly but i'm going to keep this cleanup branch in where we put a bunch of exceedingly minor small changes and update when we feel we have reached critical mass for this to be pushable.

### CODE REVIEW

File: Source/relay/TourGuide/Items of the Month/2021/Underground Fireworks Shop.ash [hunk 1]
```diff
@@ -53,7 +53,7 @@ void IOTMUndergroundFireworksShopGenerateResource(ChecklistEntry [int] resource_
 		{
 			string [int] description;
 			description.listAppend("Can buy one of the following (500 meat):");
-			description.listAppend("Fedora-mounted mountain: +20 ML hat");
+			description.listAppend("Fedora-mounted fountain: +20 ML hat");
 			description.listAppend("Sombrero-mounted sparkler: +5% combat hat");
 			description.listAppend("Porkpie-mounted popper: -5% combat hat");
 			resource_entries.listAppend(ChecklistEntryMake("__item fedora-mounted mountain", "clan_viplounge.php?action=fwshop&whichfloor=2", ChecklistSubentryMake("Dangerous hats", description), 8).ChecklistEntrySetIDTag("Clan fireworks hat resource"));
```
Comment (ajcoppa) [inline Source/relay/TourGuide/Items of the Month/2021/Underground Fireworks Shop.ash:59]: Gotta change this one too for the image to be right

Comment (docrostov) [reply Source/relay/TourGuide/Items of the Month/2021/Underground Fireworks Shop.ash:59]: yep, gotta do that. will do after today's run

**Comment by docrostov** [comment]
image
persistent annoying error to fix

**Comment by docrostov** [comment]
i couldn't figure out why the logic isn't working, so instead i had it print # of lianas left whenever it generates the "Fight the protector spirit!" tile. did fix the mountain typo too.

i will say, after playing with TES' orb tile, i remain unconvinced it is good to have this tied to the top of the screen for like 50% of the run. but i'm annoyed the tile itself has been broken for such a long time, so i think we probably merge it and just sort of hope people aren't annoyed about it. it does kind of make me want to figure out a way to make a pref-based system that swaps task vs resource assignment for a variety of tiles

**Comment by docrostov** [comment]
have tested and will merge up and into main/release

**Pull request merged**